src/nativeAssetManager.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Handles postMessage requests and responses for replacing native placeholder
+ * values in native creative templates.
+ */
+
+const NATIVE_KEYS = {
```

Comment:
might be a good idea to point to the corresponding prebid.js source in a comment here.